pages/windows/dir.md

```diff
@@ -0,0 +1,19 @@
+# DIR 
+
+> Displays file and folders in current directory or given directory.
+
+- Show list of files and folder of current directory:
+
+`dir`
+
+- Show list of file and folder in any other directory:
```

Comment:
in any other directory => in a given directory

file => files

folder => folders